sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -859,13 +859,14 @@ def _add_argparse_args(cls, parser):
             'subnetwork name. For more information, see '
             'https://cloud.google.com/compute/docs/vpc/'))
     parser.add_argument(
+        '--sdk_container_image',
```

Comment:
Any reason to use `sdk_container_image` not `sdk_harness_container_image`? The term `sdk_harness_container` is already used in the option `--sdk_harness_container_image_overrides` below. I think it would be better to use the same one for the both options (either `sdk_harness_container_image` here or `sdk_container_image_overrides` there).

Reply:
"harness" doesn't convey any additional meaning (and sometime even causes confusion). We could add sdk_container_image_overrides as an option there.